src/tools/rbd/Utils.cc

```diff
@@ -1085,5 +1086,30 @@ int mgr_command(librados::Rados& rados, const std::string& cmd,
   return 0;
 }
 
+bool image_cache_parse(const std::string& s, ImageCacheState &cache_state) {
```

Comment:
Nit: no need for this to be in a common utility file since it's only used by `Status.cc` -- just put it in that file within an anonymous namespace

Reply:
Moved to Status.cc.